src/ServiceInsight.Tests/packages.config

```diff
@@ -5,6 +5,7 @@
   <package id="Caliburn.Micro.Core" version="2.0.2" targetFramework="net45" />
   <package id="NSubstitute" version="1.10.0.0" targetFramework="net45" />
   <package id="NUnit" version="3.2.1" targetFramework="net45" />
+  <package id="ObservablePropertyChanged" version="0.1.3" targetFramework="net452" developmentDependency="true" />
```

Comment:
@adamralph I tried updating the Resharper to see if it goes away but it didn't. These warnings also would show up in TeamCity if we turn on the inspections (which not surprisingly is disabled in SI build).

I think the only way to utterly mute it would be to add the exclusion of this specific warning to .DotSetting file.